src/Framework/AspNetCoreAnalyzers/src/Analyzers/Infrastructure/WellKnownTypeData.cs

```diff
@@ -0,0 +1,203 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.App.Analyzers.Infrastructure;
+
+internal static class WellKnownTypeData
+{
+    public enum WellKnownType
```

Comment:
Or can we just go back to not nesting this in another class so we can get rid of all the extra `using WellKnownType = WellKnownTypeData.WellKnownType;` lines?